public/app/core/components/AccessControl/Permissions.tsx

```diff
@@ -102,23 +102,23 @@ export const Permissions = ({
     () =>
       sortBy(
         items.filter((i) => i.teamId),
-        ['team']
+        ['team', 'isManaged']
```

Comment:
question: what difference does this change make?

Reply:
Non managed permissions will be sorted before managed and if we have equal action length the non managed will be displayed